tutorials/docker-run.mdx

```diff
@@ -0,0 +1,567 @@
+
+So, when determining the replica count, it’s essential to consider both system capacity and reliability. Generally, we
+recommend using **at least 3 replicas for internal testing** and **5 or more for production environments.** Increasing
+the number of replicas can significantly enhance the reliability and throughput of your applications on SaladCloud.
+
+<img src="/tutorials/images/docker_run/docker_run6.png" />
+
+**Deploying a container group with a few instances may also reduce waiting time and enhance the developer experience
+without incurring additional costs.**
+
+For example, if you want to conduct a quick test or validate an idea on SaladCloud, you can deploy a container group
+with 3 replicas. Since some nodes may start up earlier than others, once one of the replicas is running, you can perform
+and complete your test, and then shutdown the group, potentially before the other two replicas are up. Throughout this
+process, you would only be charged for the running time of a single replica.
+
+### Machine Hardware
+
+The --gpus all option in the command allocates all available GPUs on the host to the container, enabling GPU support for
+applications running inside it. SaladCloud currently supports only one GPU per container. When creating a container
+group, you can select multiple GPU types at a priority level [(Batch/Low/Medium/High)](/products/sce/priority-pricing),
+along with the required vCPU, memory and disk space. SaladCloud will then allocate the appropriate Salad nodes that
+match these selections to run the container group.
+
+<img src="/tutorials/images/docker_run/docker_run7.png" />
+
+Setting a lower priority for the selected GPUs can help reduce costs; however, low-priority workloads may be preempted
+when higher-priority workloads become available on SaladCloud. If you notice that instances with a lower priority level
+are being reallocated too frequently for your needs, you can edit the running container group to select a higher
+priority level.
+
+### Salad Job Queue
+
+If you are using the Salad Job Queue to manage the tasks for your container group, you can use Salad Job Queue Worker
+(an executable binary) to replace io_worker.py in the example. This worker is designed to handle input/output operations
+and interact with your inference server. For more information about Salad Job Queue, please refer to
+[this link](/products/sce/job-queues).
+
+### Substitutes
+
+Not all parameters in the docker run command can be directly converted to a container group deployment on SaladCloud;
+however, alternative solutions are available.
+
+The -it flag allows you to interact with a container through a terminal. While SaladCloud does not offer this option
+during container group creation, you can still interact with running instances using
+[the interactive terminal](/tutorials/howtousethesaladterminal) after the group has been created to perform tests or
```

Comment:
yeah, let's rename this page instead, and work toward consistent hyphen-separated-phrases when we have multi-word filenames.